cdci_data_analysis/analysis/instrument.py

```diff
@@ -63,6 +63,18 @@
 # TODO: this is not preserved between requests, and is not thread safe. Why not pass it in class instances?
 params_not_to_be_included = ['user_catalog',]
 
+general_use_args = ['instrument', 
```

Comment:
word "general" is to general. if there is no less general word, could you please add a comment above explaining it's use?

Reply:
possibly "non_parameter_args"?